script/old/upload_observation.rb

```diff
@@ -515,7 +515,7 @@ def post_data
 
   def post_observation
     http = Net::HTTP.new(HOST, PORT)
-    path = make_path("/api/observations",
+    path = make_path("/api2/observations",
```

Comment:
@pellaea Are the scripts in `script/old` used?
If yes, what's intended here (`api` vs `api2`)?